### PR TITLE
Add 5.1 audit report

### PR DESCRIPTION
**Pull request by ernestognw**
We may wait until the final 5.1 version is published to add the version commit 

#### PR Checklist

- [ ] Tests
- [ ] Documentation
- [ ] Changeset entry (run `npx changeset add`)

### CODE REVIEW

**Review by Amxx — APPROVED** [review]

**Comment by Amxx** [comment]
or should we merge that and then update the commit hash ?

File: audits/README.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 | Date         | Version | Commit    | Auditor      | Scope                | Links                                                       |
 | ------------ | ------- | --------- | ------------ | -------------------- | ----------------------------------------------------------- |
+| October 2024 | v5.1.0  | TBD       | OpenZeppelin | v5.1 Changes         | [🔗](./2024-10-v5.1.pdf)                                    |
```
Comment (ernestognw) [inline audits/README.md:5]:
```suggestion
| October 2024 | v5.1.0  | TBD (RC)  | OpenZeppelin | v5.1 Changes         | [🔗](./2024-10-v5.1.pdf)                                    |
```

**Review by ernestognw — COMMENTED** [review]
Let's specify it's an ERC and we come back to edit if needed, wdyt @Amxx ?

File: audits/README.md (hunk 1 above)
Comment (ernestognw) [reply audits/README.md:5]: Not relevant. Let´s merge it.

**Pull request merged**